extension-style/build.gradle.kts

```diff
@@ -23,6 +23,7 @@ dependencies {
   implementation(Dependencies.androidxAppCompat)
   implementation(Dependencies.androidxCoreKtx)
   implementation(Dependencies.androidxAnnotations)
+  implementation(Dependencies.coroutines)
```

Comment:
Just curious - how does this affect binary size?

Reply:
The release aar size for extension-style on master is 612kb, on this branch is 613 kb.

Reply:
I guess the aar doesn't contain the binary size of coroutine dependency 🙂